Failing to compile:  error[E0283]: type annotations needed

Compiling from `cargo install kbt` `kbt = "2.0.6"` and from this cloned repo yields:

```rust
error[E0283]: type annotations needed
  --> src\view.rs:42:10
   |
42 |         .constraints(make_row_constraints(row_keys).as_ref())
   |          ^^^^^^^^^^^                                ------ type must be known at 
this point
   |          |
   |          cannot infer type of the type parameter `C` declared on the method `constraints`
   |
   = note: multiple `impl`s satisfying `Vec<Constraint>: AsRef<_>` found in the `alloc` crate:
           - impl<T, A> AsRef<Vec<T, A>> for Vec<T, A>
             where A: Allocator;
           - impl<T, A> AsRef<[T]> for Vec<T, A>
             where A: Allocator;
help: consider specifying the generic argument
   |
42 |         .constraints::<&T>(make_row_constraints(row_keys).as_ref())
   |                     ++++++

For more information about this error, try `rustc --explain E0283`.
error: could not compile `kbt` (bin "kbt") due to previous error
PS C:\Users\Brioche Elm\Documents\GitHub\kbt> 
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,7 +12,7 @@ readme = "README.md"
 
 [dependencies]
 device_query_revamped = "2.2.1"
-ratatui = { version = "0.24.0", features = ["all-widgets"]}
+ratatui = { version = "0.25.0", features = ["all-widgets"]}
 crossterm = "0.27"
 clap = { version = "4.4.7", features = ["derive"] }
 simple-logging = "2.0.2"

--- a/src/view.rs
+++ b/src/view.rs
@@ -39,7 +39,7 @@ pub fn draw(frame: &mut Frame, state: &App) -> Result<(), KbtError> {
 fn draw_row(row_keys: &[KeyUI], state: &App, keyboard_rect: Rect, frame: &mut Frame) {
     let chunks = Layout::default()
         .direction(Direction::Horizontal)
-        .constraints(make_row_constraints(row_keys).as_ref())
+        .constraints(make_row_constraints(row_keys))
         .split(keyboard_rect);
 
     for (x_pos, ui_key) in row_keys.iter().enumerate() {

